abcclassroom/clone.py

```diff
@@ -99,17 +99,58 @@ def clone_student_repos(args):
         print(err)
 
 
-def copy_assignment_files(config, student, assignment_name):
+def include_patterns(patterns):
```

Comment:
@nkorinek i am working on the clone module in #318 so just be careful here but this definitely should go in utils.